components/Heroes/BenchmarkBadge.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Grid, Row, Col } from 'react-flexbox-grid';
```

Comment:
writing media queries need lots more of time than using react-flexbox-grid. iirc, my idea was because someone on discord has problem accessing data from mobile, so I did something diffrent to minimalize this problems by using flexbox. Also, react-flexbox-grid isn't for table components.